content/pages/roadmap.html

```diff
@@ -16,61 +16,98 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
+                    <div>
+                        <p>
+                        This approach is about cleaning up the design of the original protocol. From a functionality standpoint, interacting with iRODS would remain nearly identical. The primary difference would be in the design and implementation. The primary benefit of this is that it provides freedom in the sense that it can changed.
+                        </p>
 
-                <h3>Filesystem Integration</h3>
+                        <p>The new protocol must satisfy the following requirements:</p>
+                        <ul>
+                            <li>Documented</li>
+                            <li>Versioned</li>
+                            <li>Improved error handling</li>
+                            <li>Extendable</li>
+                            <li>Retry mechanisms</li>
+                        </ul>
 
-                <p>iRODS strives to serve as the glue that can tie together many existing storage technologies with a unified namespace, discovery mechanism, and policy engine.  Usually, when an organization decides they need to incorporate iRODS into their infrastructure, they already have a significant amount of data, usually in disparate physical systems.</p>
+                        <p>One area that requires careful consideration is whether the protocol should be stateless or stateful. Perhaps it can support both.</p>
+                    </div>
 
-                <p>iRODS needs a way to keep track of changes occurring on filesystems, where existing scripts, tools, and instruments are making changes outside of iRODS.  This can be done for moderate-sized systems with a pattern we have named the "Landing Zone" whereby iRODS watches a particular location in the filesystem and takes action when new files appear.  For much larger systems handling a very large number of files concurrently, this approach may become an unreasonable bottleneck.</p>
+                    <h3>New Plugin Interface</h3>
+                    <div>
+                        <p>This approach is more complex, but offers the most flexibility to developers.</p>
 
-                <p>We are working with Intel's Lustre group to investigate and then implement best practices around keeping the iRODS catalog up-to-date for such high-velocity systems.</p>
+                        <p>Introducing a plugin interface for protocols means the number of protocols supported by iRODS is unbounded. However, this requires expert level design skills and a full understanding of all components involved with iRODS and networking.</p>
 
-                <h3>Metadata Abstraction</h3>
+                        <p>This effort must satisfy the same requirements defined under <a href="#new-custom-protocol">New Custom Protocol</a>. Ultimately, this is a final resort. Given that the focus of 4.4 is simplification and maintainability, we prefer to keep the number of protocols at one.</p>
```

Comment:
oh, i see the link to the earlier anchor now.

Reply:
Do we want to restore the anchor?

Reply:
no, the document isn't that heavy/long.   let's keep them text within each section...

Reply:
Ok